functions/venue.js

```diff
@@ -842,7 +842,19 @@ exports.updateVenueNG = functions.https.onCall(async (data, context) => {
   // @debt updateVenue uses checkUserIsOwner rather than checkUserIsAdminOrOwner. Should these be the same? Which is correct?
   await checkUserIsOwner(data.id, context.auth.token.user_id);
 
-  const updated = {};
+  if (!data.worldId) {
+    throw new HttpsError(
+      "not-found",
+      "World id is missing and the update can not be executed."
```

Comment:
[NOTE]
We shouldn't send user facing messages from BE, but since it is the case, I guess `Id` is the "proper English" spelling

Reply:
I guess I can remove it, but If world id is missing this function will probably return nothing, and look weird on the client side.

Reply:
This is more like a note for a longer term consideration